### ISSUE TITLE
can't find in HACS

### ISSUE THREAD

**Issue by ktownsend-personal** [xannor/hass_py_set_state#3]
I have been using this for quite a while and noticed it is supposed to be available in HACS, so I decided to switch to the HACS managed install of it. For some reason I'm not finding it in HACS. I found something similar from someone else in HACS ([here](https://github.com/pmazz/ps_hassio_entities)), but I really just wanted to use yours. Is there a reason yours isn't available in HACS?

**Comment by xannor** [comment]
I have never requested for it be added to HACS, it just supports it. I'll look into what it takes to get it added.

**Comment by ktownsend-personal** [comment]
I just figured out that there is a way for me to add your repo to my HACS myself. I had no idea that option was there! I never noticed the "..." button at the top right of HACS before. I've never noticed anyone's HACS install steps in their readme.md mention manually adding the repo, so I just assumed it would already be in the list and never that was a feature. Sorry about my confusion on that.

Incidentally, I found this page on HACS with instructions to get added to the default list if you want to do that: https://hacs.xyz/docs/publish/include
